### ISSUE TITLE
Move sharing and archiving to communication and add info about zenodo release

### ISSUE THREAD

**Issue by malvikasharan** [alan-turing-institute/the-turing-way#1786]
### Summary

* The chapter on sharing and archiving exists in RDM at the moment: https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-sharing.html

### What needs to be done?

- [ ] Move it to the communication guide
- [ ] Add links to Zenodo and figshare
- [ ] Write a paragraph on how to connect GitHub and Zenodo for release

### Who can help?

* I will work on it tomorrow
* The chapter is drafted by @EstherPlomp who may want to add any info

---

**Comment by EstherPlomp** [comment]
Hi @malvikasharan ! I'm very confused why this chapter should be moved to the communication guide? Sharing data is not just about communication but also about long term preservation, so I don't think anyone would look there for this information? I could see it being moved to the open research chapter?

**Comment by malvikasharan** [comment]
I will be running a workshop of communication and collaboration soon and I would like to share a chapter that gives an overview of how to prepare to share your research data and code. I can write a similar chapter without moving it that covers both data and code, cross referencing to this chapter.

**Comment by EstherPlomp** [comment]
Or if the code part isn't extending the chapter too much it could be added here rather than moving it? It might indeed be better to have a new chapter on it because the current one is already quite large!

**Comment by EKaroune** [comment]
I think a slightly different rewrite for the communication guide would be the best move - about why to share data/code and link to how to do it, could also have a case study at the end showing a good example?

**Comment by KirstieJane** [comment]
Heya! Thanks for this discussion @EstherPlomp & @malvikasharan! This issue came from a conversation with me so I'll try to explain a little more here.

> Hi @malvikasharan ! I'm very confused why this chapter should be moved to the communication guide? Sharing data is not just about communication but also about long term preservation, so I don't think anyone would look there for this information? I could see it being moved to the open research chapter?

I think there's a mix up between the _guides_ and the _chapters_ in this reply. *Communication* and *Reproducibility* are guides. Importantly, *Reproducibility* has a lot of content because it was the only guide in the first place... there are quite a lot of chapters in there that should ideally be moved to the expanded guides.

The problem with RDM is that it's something that fits across all of the parts of a research project lifecycle and I think what I'd like to propose is that we actually split up the current chapter & fit the components into the 5 guides as they exist.

To be honest, I think a lot of RDM should move to Project Design, with a couple of points going into Communication!

Here's my first pass proposal going through the different sub-chapters that are in the RDM chapter at the moment.

* Research Data -- Renamed to "Overview?" -- Project Design -- I feel like I'd put this here to direct people to thinking about _why_ they need to manage their data and set it up to follow a standard etc. Linking very explicitly to both reproducibility but also reusability.
  * https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-data.html
* Data Management Plan -- Project Design -- your likely engagement with a DMP is when you're _starting_ a project (even though it lasts for the whole of it)
  * https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-dmp.htmlThe FAIR Principles and Practices
* The FAIR Principles -- Project Design?? This one I struggle with a little. It might actually be part of Collaboration??? Like, WHY do we want data to fit with these principles? Open to suggestions on that one.
  * https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-fair.html
* Data Storage and Organisation -- Project Design -- again, feels like things you're going to mostly think about at the start of a project
  * https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-storage.html
* Data Organisation in Spreadsheets -- Project Design I think?? But maybe Reproducibility -- open to suggestions here! These are like "tips and tricks for working effectively" so it sits across those two.
  * https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-spreadsheets.html
* Documentation and Metadata -- this one I think starts to connect Project Design and Communication. You should be documenting your work and adding metadata so that you can communicate the work to others...
  * https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-metadata.html
* Sharing and Archiving Data -- **Communication** -- it should be linked from Project Design but its says in the name: its how to share your work and make it available for others in the long term
  * https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-sharing.html
  * 👉 Here's where the zenodo update would go as a sub chapter. But its important that it stand at least a little apart because we want to encourage sharing slides etc not just data.
* Research Data Management Toolkits -- Project Design, or wherever data organisation in spreadsheets lives?
  * https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-toolkits.html
* Personal Impact Stories -- This needs to sit with the FAIR Principles and have a different title to make it clear that its a story related to that topic. (Maybe sub-sub chapter, is that a thing?)
  * https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-stories.html
* Checklist -- personally I'd move this to overview (above)
  * https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-checklist.html
* Resources -- I think I'd move this to overview too
  * https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-resources.html

PHEW - that took me ages multitasking in a few meetings - I hope it makes sense!

**Comment by KirstieJane** [comment]
Yes - super up for a conversation about it! Please do bring in a few more (although @malvikasharan - this might be better in a new issue - I might have derailed a bit!)

**Comment by Karvovskaya** [comment]
Hi there, 
I've been looking at this chapter because one of the wishes from my team is to have a visualization explaining the difference between STORAGE and ARCHIVING. While reading the chapter I actually found little specifically on archiving (if not for sharing). Is it something to add? @EstherPlomp what do you think? Also tagging @Elisa-on-GitHub

**Comment by jcolomb** [comment]
Archiving is indeed different from publication (that is sharing), you usually do one or the other. It makes sense to have it together (as published data do not need extra archiving), and archiving has nothing to do with communication.

There will always be different ways to navigate the book... so difficult to find the best place for the content, often depends on the story you tell and the point of view / interest you have.

**Comment by jcolomb** [comment]
I am wondering whether it would not be useful to use a mind mapping tool like miro to discuss the organisation of this content and move things around virtually there first ?

**Comment by Karvovskaya** [comment]
Good idea @jcolomb! Mind mapping should be good for this!

**Comment by Karvovskaya** [comment]
I see it's a very old issue. Maybe I should not have touched it. I was primarily interested in finding a place for Archiving 😄

**Comment by jcolomb** [comment]
While a one year old issue, it makes total sense to tackle this now. I have worked a mindmap a bit, using the [miro mentioned above ](https://miro.com/welcomeonboard/VzN5R3NNMkhvU2R6czZGbEFXNDZTWEhGSzNaQWdFbjhzcTM3NU9Ya1QyMEFWa0RJQzRUT0dSVWZoZFA2MVVRMnwzMDc0NDU3MzQ4NDYzNjU0NzEx?share_link_id=680078991357).

For many aspects of RDM, the same topic would fit in nearly all guides. For instance version control is needed for provenance and therefore for reproducibility, it needs to be included in the project planning as it has effects on the cost of storage and can even have influence on the type of data format one uses, it is primordial to make collaboration on the data possible, and it may even be necessary for communication, as different versions of the data might need to be archived/published, and that the version control tool choice will make publication more or less easy.

I tend to think it fits best all in the reproducibility guide, but that we may want to check that the other guide do link (and not repeat) what is in the reproducibility guide. Another possibility would be to have these subchapter added to several guides (would get the guides independent from each other, would need to be sure that the subchapters are written accordingly or have a special mention at the start stating they were originally meant for the reproducibility guide)

**Comment by jcolomb** [comment]
Proposition: for each RDM chapter , write/copy-paste a `planning` subchapter and an `implementation` subchapter, put them In the different guides, make sure they reference each other.
Plus give the possibility to have it all one after the other in a different approach using https://deploy-preview-2246--the-turing-way.netlify.app/welcome.html#different-pathways

**Comment by jcolomb** [comment]
Created a new issue, I propose to close this one.
